### DISCUSSION TITLE
Terrain using multiple .usd files

### DISCUSSION THREAD

**Discussion by SourabhPrasad** [isaac-sim/IsaacLab#1396] [Q&A] [answered]
Hi,
I want multiple USD files as terrains, but the `TerrainImporter` only accepts one USD file path. Can multiple USDs be provided to the `TerrainImporter`, or can multiple meshes be combined into a single mesh programmatically?

**Comment by StrainFlow** [comment, marked as answer]
I haven't done this myself, but looking at the documentation here: https://isaac-sim.github.io/IsaacLab/main/source/api/lab/omni.isaac.lab.terrains.html#omni.isaac.lab.terrains.TerrainImporter.import_usd

It looks like each time you import a USD file as the terrain it will only import the first mesh in the USD scene. From there, each imported terrain is added to a key value dictionary, so it seems like you should be able to call 'import_usd' more than once.
